aws_lambda_powertools/utilities/data_classes/vpc_lattice.py

```diff
@@ -101,3 +105,7 @@ def get_header_value(
             default_value=default_value,
             case_sensitive=case_sensitive,
         )
+
+    def header_serializer(self) -> BaseHeadersSerializer:
+        # When using the VPC Lattice integration, we just have single header.
+        return SingleValueHeadersSerializer()
```

Comment:
Acording to [this](https://docs.aws.amazon.com/vpc-lattice/latest/ug/lambda-functions.html#multi-value-headers), VPC Lattice Lambdas should support returning multiple headers, so I believe the correct class here is `HttpApiHeadersSerializer`.

Reply:
Great catch, Ruben! Fixed.